src/Compilers/CSharp/Test/Syntax/Parsing/PatternParsingTests.cs

```diff
@@ -11962,6 +11965,16 @@ void verifyTree()
                                                     }
                                                 }
                                                 N(SyntaxKind.CommaToken);
+                                                M(SyntaxKind.Subpattern);
+                                                {
+                                                    M(SyntaxKind.ConstantPattern);
+                                                    {
+                                                        M(SyntaxKind.IdentifierName);
+                                                        {
+                                                            M(SyntaxKind.IdentifierToken);
+                                                        }
+                                                    }
+                                                }
```

Comment:
this is technically more correct.  we have a positional pattern with a trailing comma.  We should have a final missing subpattern.  The old code would not put one in since it vailed on `}`.  but we only want that bailing for *property* patterns, not positinoal ones.